docs/utilities/oredictionary.md

```diff
@@ -0,0 +1,77 @@
+OreDictionary
+======
+
+The OreDictionary exists primarily for inter-mod compatibility. 
+
+By registering ores to the OreDictionary, mods make it possible to use the ores and ingots from another, similar mod in their recipes.
+
+Despite its name, the OreDictionary is used for much more than ores. Any item that is similar to another item (for example, dyes) can be registered to and used with the OreDictionary.
+
+## OreDictionary Name Convention
+
+!!! note
+    Because OreDictionary names are meant to be shared between items from different mods, they should be fairly general. Use a name that other mods are likely to use, based on these conventions.
+	
+All OreDictionary names should be camelCase (compound words that begin with a lowercase letter, where each successive word begins with a capital letter). Avoid spaces or underscores.
+
+The first word in the OreDictionary name indicates the type of item. For unique items (such as `record`, `dirt`, `egg`, and `vine`), one word is specific enough.
+
+The last part of the name indicates the material of the item -- for example, `Wood` or `Iron`.
+
+When two words are not specific enough, a third word can be added. For instance, brick is registered as `ingotBrick` while nether brick is registered as `ingotBrickNether`.
+
+See [Common OreDictionary Names](#common-oredictionary-names) for a list of commonly used prefixes and suffixes.
+
+## WILDCARD_VALUE
+
+This value is used to indicate that the metadata of an `ItemStack` is not important. See below for an example of its use.
+
+## Using OreDictionary in Crafting Recipes
```

Comment:
In this section, avoid using chunks of code. Mentioning the recipe classes and their vanilla counterparts should be enough. Use small snippets of code only if it actually improves the information.